es-ds-docs/pages/molecules/accordion.vue

```diff
@@ -0,0 +1,499 @@
+<script setup lang="ts">
+const programmaticExpandedId = ref('programmatic-question-1');
+
+const accordionListProps = [
+    [
+        'allowMultipleExpand',
+        'false',
+        `
+        Defaults to false, which only allows one item to expand at a time. If true, allows multiple
+        items to expand at a time. Do not use at the same time as v-model.
+        `,
+    ],
+    [
+        'initialExpandedId',
+        '',
+        `
+        The id of the item that should begin expanded when the component mounts. If omitted, all
+        items will start out collapsed. Do not use at the same time as v-model.
+        `,
+    ],
+    [
+        'variant',
+        'default',
+        `
+        Defaults to 'default'; also accepts 'minimal'. Default has a rounded white background,
+        minimal has no background, Heading 3 text style, reduced padding, and a horizontal border
+        separating accordions.
+        `,
+    ],
+];
+const accordionProps = [
+    [
+        'id',
+        '',
```

Comment:
Should we put n/a for props with no default value? I saw it done in at least 1 other place

Reply:
I'm not sure.. does that run the risk of people thinking that "N/A" is the default value? I figure a blank space means there's no default value

Reply:
Looking at this with a fresh set of eyes I think you're right, and it's no big deal in any case